### ISSUE TITLE
Firefox warns about properties of the cookie "refreshToken"

### ISSUE THREAD

**Issue by hanak** [wduckitt/React-Automation-Studio#29]
**Problem**
Login/logout sequence yields following warnings in the console of the browser (Firefox):
* Cookie “refreshToken” will be soon rejected because it has the “SameSite” attribute set to “None” or an invalid value, without the “secure” attribute. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite
* Cookie “refreshToken” has been rejected because it is already expired.

**Comment by wduckitt** [comment]
Hi, 
Is this not an incorrect reporting of an issue in Firefox.
If I inspect the cookie flags in Firefox: 
HttpOnly is true
Secure is true
and Samesite is none

Firefox should not print the warning mentioned above in my opinion or am I missing something.
Firefox version: 89.01

No issues are reported in Chrome

**Comment by hanak** [comment]
Thank you for your answer. I have seen cookie-related reporting on Firefox only. Yet, it seems I was creating the issue in hurry and I forgot to add details. I am sorry:
* Firefox version: 89.0.1
* Secure: false (using HTTP)

Furthermore, I cleared everything I could. And, I tried it again. And, it looks like `/api/logout` sets a cookie regardless `SECURE` while `/api/login` avoids adding a cookie if `SECURE` is `False`. Also, the cookie sent at `/api/logout` does not include attributes which Firefox is complaining about. Yet, I understand the intention here is to remove the cookie. If the behavior of `/api/logout` is expected in this context, we can close this issue.
